src/scss/skin-modern/components/_controlbar.scss

```diff
@@ -7,6 +7,8 @@
   @include hidden-animated;
   @include layout-align-bottom;
 
+  visibility: unset !important;
```

Comment:
`!important` is a no go and brings our customers a lot of trouble. Please find a better way.

Reply:
The `visibility` property removes the current focus location and position of navigation.
As seen in the usage of other players they are just changing the opacity of the elements and never hiding them. In that way the focus is still kept on the current element and you are able to manipulate with that element with the keyboard.
I can't unfortunately think of any other way of maintaining the focus on the current element when the controls are `hidden`.